workers/unity/Packages/com.improbable.gdk.transformsynchronization/MonoBehaviours/TransformSynchronization.cs

```diff
@@ -94,6 +94,7 @@ private void ApplyStrategies()
             if (SetKinematicWhenNotAuthoritative)
             {
                 commandBuffer.AddComponent(entity, new ManageKinematicOnAuthorityChangeTag());
+                commandBuffer.AddComponent(entity, new KinematicStateWhenAuth());
```

Comment:
Is this a driveby bugfix?

Reply:
Yes. Noticed when testing that nothing happened with the kinematic state. Eventually found out I'd not wired it up.